app/lib/data.js

```diff
@@ -68,49 +68,49 @@ export default {
                   }
                ]
             },
-            {  
-               "id":102,
-               "checkDesc":"Miniature Games",
-               "basePrice":0,
+          ]
+      },
+      {
+         "id":102,
```

Comment:
I updated the input data to make 102 a sibling of 101, as per clarified requirements